app/assets/javascripts/admin/addon/components/emoji-value-list.js

```diff
@@ -47,23 +46,19 @@ export default Component.extend({
 
   _splitValues(values) {
     if (values && values.length) {
-      const res = [];
+      const resultEmojiList = [];
       const emojis = values.split("|");
-      emojis.forEach((emoji, index) => {
-        const object = {};
-        object.value = emoji;
-
-        object.emojiUrl = emojiUrlFor(emoji);
-        object.isEditable = true;
-
-        object.isEditing = false;
-
-        object.isLast = emojis.length - 1 === index;
-
-        res.push(object);
+      emojis.forEach((emoji) => {
```

Comment:
maybe you can even go with emojiNames here, it will allow you to use emojis for the list

Reply:
used `emojiList` `emojis` `emojiName` `emoji` in the latest commit

Reply:
please  group by change and give better names to your commits: "changes as per comments" is not a good name, even if temporary, it makes it harder for me to review and understand what you changed  each time.

Reply:
Oké ill keep this in mind😅